R/resource_patch.R

```diff
@@ -14,14 +14,14 @@
 #' res$description
 #'
 #' # Make some changes
-#' x <- list(description = "My newer description")
+#' x <- list(description = "My newer description", "extra_key" = "my special value")
```

Comment:
this change looks good, but can you change the entire example block here to make a package, make a resource, then patch it, e.g., 

```r
#' # create a package
#' (res <- package_create("twist", author="Alexandria"))
#'
#' # then create a resource
#' file <- system.file("examples", "actinidiaceae.csv", package = "ckanr")
#' (xx <- resource_create(package_id = res$id, description = "my resource"))
#' 
#' # Get a resource
#' res <- resource_show(xx$id)
#' res$description
#'
#' # Make some changes
#' x <- list(description = "My newer description", "extra_key" = "my special value")
#' z <- resource_patch(x, id = res)
#' z$description
```

Reply:
sure